app/_components/screenshots/template.njk

```diff
@@ -1,17 +1,21 @@
 {%- from "../figure/macro.njk" import appFigure with context -%}
+{%- set title = params.title or "Screenshots" -%}
+{%- set headingLevel = params.headingLevel or 2 -%}
 <section class="app-screenshots app-prose govuk-!-margin-top-9" id="screenshots">
   {% if not params.hideContents %}
-    <h2 class="govuk-heading-l govuk-!-display-none-print">
-      <a class="header-anchor" href="#screenshots">#</a>
-      Screenshots
-    </h2>
-    <ul class="govuk-list app-screenshots__contents govuk-!-display-none-print">
+  <div class="govuk-!-display-none-print">
+    <h{{ headingLevel }}>
+      <a class="header-anchor" href="#{{ title | slug }}">#</a>
+      {{ title }}
+    </h{{ headingLevel }}>
```

Comment:
Nice :+1:

Reply:
Will merge it into generic design history once this entry is merged.